pkg/slo-controller/noderesource/resource_calculator.go

```diff
@@ -120,7 +104,8 @@ func (r *NodeResourceReconciler) updateNodeResource(node *corev1.Node, nr *frame
 func (r *NodeResourceReconciler) updateNodeExtensions(node *corev1.Node, nodeMetric *slov1alpha1.NodeMetric, podList *corev1.PodList) error {
 	// update device resources
 	if err := r.updateDeviceResources(node); err != nil {
```

Comment:
should we add a config or feature-gate for device calculation?

Reply:
It has been discussed in https://github.com/koordinator-sh/koordinator/pull/1065#issuecomment-1457451506, so we do not change the device calculation in the issue #1060.